### PR TITLE
network-diagnostics fixes

### PR DESCRIPTION
**Pull request by leodotcloud**
- https://github.com/rancher/rancher/issues/9296
- https://github.com/rancher/rancher/issues/9299

Fixing the download link
Enabling healthcheck for network-diagnostics service

### CODE REVIEW

**Comment by deniseschannon** [comment]
Can you add some min/max Rancher versions on these tools. We should only have 1 tool available to be deployed.

File: infra-templates/network-diagnostics/1/rancher-compose.yml [hunk 1]
```diff
@@ -1,5 +1,19 @@
 .catalog:
-  version: 0.0.1
+  version: 0.0.2
+  minimum_rancher_version: v1.6.5-rc1
+  services:
+  network-diagnostics:
```
Comment (deniseschannon) [inline infra-templates/network-diagnostics/1/rancher-compose.yml]: Why is healthcheck in the docker-compose and rancher-compose.yml? Pick 1 place. Also, if it's in rancher-compose.yml, you need to place it outside of the catalog section.

Comment (leodotcloud) [reply infra-templates/network-diagnostics/1/rancher-compose.yml]: ah that explains why it was not working. I am still testing this. so it's in two places. Will fix it.

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**